azure-pipelines.yml

```diff
@@ -22,6 +22,9 @@ steps:
     echo "##vso[task.setvariable variable=docker_path;]$docker_path"
   displayName: 'Set version number'
 
+- script: docker pull $(docker_path):latest || true
```

Comment:
Why do we need `|| true`? Does `docker pull` return false negatives?

Reply:
It's to ensure the first run does't fail, also ensures any non-zero return codes don't cause the whole build to fail. It just means the build won't end up using layer caching so would take a bit longer.

Reply:
I get the mechanics of `|| true`, my point is that with it if there's a failure in pulling the image we won't get an error here and it would be confusing and harder to debug. But I think you're saying that we need it because the first time we run, there may not be a `latest` so there won't be anything to pull and cache from?

Reply:
Yes that's it.